src/main/java/frc/robot/Constants.java

```diff
@@ -165,12 +170,12 @@ public static final class Intake {
     /** time in seconds to wait between certain modes */
     public static final class ModeWaits {
       public static final class High {
-        public static final double ALIGN_TO_LEFT = .24 * 2;
-        public static final double LEFT_TO_ALL = .55;
-        public static final double ALL_TO_OFF = 2;
+        public static final double ALIGN_TO_LEFT = .08;
+        public static final double LEFT_TO_ALL = .7;
+        public static final double ALL_TO_OFF = 4;
```

Comment:
Why did we change the time needed to change between modes? 

Also, wouldn't we want the time needed to change between modes to be 0? Would it be because the motors need time to change speed or something?

Reply:
This was based on tuning done sat. Time to change between modes is non-zero to stage the shot. We delay pushing the second ball forward to prevent interaction.

Reply:
ah I see